docs/.vuepress/config.js

```diff
@@ -165,12 +165,12 @@ module.exports = {
         collapsable: true,
         sidebarDepth: 1,
         children: [
-          ['/testnet-plasma', 'PlasmaChain Testnets'],
-          ['/mainnet-plasma', 'PlasmaChain MainNet'],
+          ['/testnet-plasma', 'Loom Testnet'],
```

Comment:
The title no longer matches the page title

Reply:
> The title no longer matches the page title

Right. Now it's updated.